app/src/main/java/database/ObjectBox.kt

```diff
@@ -5,11 +5,15 @@ import database.entity.MyObjectBox
 import io.objectbox.BoxStore
 
 object ObjectBox {
+    val DB_DIRECTORY_NAME = "database"
+    val DB_FILE_NAME  = "data.mdb"
+
     lateinit var boxStore: BoxStore
         private set
 
     fun init(context: Context) {
         boxStore = MyObjectBox.builder()
+            .initialDbFile(DatabaseStreamFactory(context.assets.open(DB_DIRECTORY_NAME+"/"+DB_FILE_NAME)))
```

Comment:
You can use Kotlin's string interpolation here to make it a bit nicer

"${DB_DIRECTORY_NAME}/${DB_FILE_NAME}"

Reply:
you can name it whatever you want, but that's the default ObjectBox name so I didn't bother changing it